rotkehlchen/db/db_cache.py

```diff
@@ -0,0 +1,74 @@
+from enum import auto
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.types import Timestamp
+from rotkehlchen.utils.mixins.enums import Enum
+
+
+DBCacheFieldTypes = int | Timestamp
+
+
+class DBCache(Enum):
+    """It contains all the values that can be stored in the `key_value_cache` table of the DB"""
+    LAST_WRITE_TS = auto()
+    LAST_BALANCE_SAVE = auto()
+    LAST_DATA_UPLOAD_TS = auto()
+    LAST_DATA_UPDATES_TS = auto()
+    LAST_OWNED_ASSETS_UPDATE = auto()
+    LAST_EVM_ACCOUNTS_DETECT_TS = auto()
+    LAST_SPAM_ASSETS_DETECT_KEY = auto()
+    LAST_AUGMENTED_SPAM_ASSETS_DETECT_KEY = auto()
+    SPAM_ASSETS_VERSION = auto()
```

Comment:
Aren't all these `_VERSION` keys charactherizations of the DB? They are not part of the cache. I don't see why you need them to be here.

Reply:
you didn't answer this one

Reply:
Yes, I've removed them from the cache now.